extensions/google-cloud-functions-http/runtime/src/test/java/io/quarkus/gcp/functions/http/QuarkusHttpFunctionTest.java

```diff
@@ -42,6 +42,8 @@ public class QuarkusHttpFunctionTest {
     private static final String HOST = "localhost";
     private static final String METHOD = "GET";
 
+    private static final String USER_AGENT = "User-Agent";
```

Comment:
I suppose you initially wanted to add a test?

Reply:
I'm writing the test, just to confirm if this in the right way.